### ISSUE TITLE
how to extracting the features?

### ISSUE THREAD

**Issue by aksahveoffice** [drprojects/superpoint_transformer#10]
i am not getting feature extraction code.

**Comment by drprojects** [comment]
Hi, please provide a more detailed explanation of the issue you are encountering.

**Comment by drprojects** [comment]
Taking a wild guess: I am assuming you would like to know where to find the extracted superpoint features. If so, a brief investigation of the code should have lead you [here](https://github.com/drprojects/superpoint_transformer/blob/6b9ac9aa0c96d843af7f50448a3fbf968263d56a/src/models/segmentation.py#L102). The output of `self.net` are the extracted features, before calling `self.head` on them to produce semantic segmentation logits.

NB: The above-mentionned features are superpoint-wise features and not point-wsie features. For more details on that your can read our [paper](https://arxiv.org/abs/2306.08045) and check out the code documentation and comments.

Without further details from you, I am closing this issue.
